test/python/transpiler/test_instruction_durations.py

```diff
@@ -88,3 +60,113 @@ def test_fail_if_get_unbounded_duration_with_unit_conversion_when_dt_is_not_prov
         parameterized_delay = Delay(param, "s")
         with self.assertRaises(TranspilerError):
             InstructionDurations().get(parameterized_delay, 0)
+
+
+class TestInstrctionDurationsFromBackendV1(QiskitTestCase):
+    """Test :meth:`~.from_backend` of :class:`.InstructionDurations` with
+    :class:`.BackendV1`"""
+
+    def setUp(self):
+        super().setUp()
+
+        self.backend = FakeParis()
+        self.backend_config = self.backend.configuration()
+        self.backend_props = self.backend.properties()
```

Comment:
```suggestion
        self.backend_config = self.backend.configuration()
        self.backend_props = self.backend.properties()
```
These two variables are only used by a particular test. In this case you can write them in individual test case rather than in the setup.

Reply:
I just tried to make a framework, in case someone adds a new test in the future :)